boards/posix/nrf52_bsim/doc/board.rst

```diff
@@ -0,0 +1,186 @@
+
+.. _nrf52_bsim:
+
+NRF52 simulated board (BabbleSim)
+#################################
+
+.. contents::
+   :depth: 1
+   :backlinks: entry
+   :local:
+
+
+Overview
+********
+
+This is a simulated NRF52 board which uses `BabbleSim`_ to simulate the radio
+activity.
+This board models some of the NRF52 SOC peripherals:
+
+* Radio
+* Timers
+* Real time counter
+* Random number gerator
+* AES CCM & AES ECB encryption HW
+* Accelerated address resolver
+* Clock control
+* PPI (Programmable Peripheral Interconnect)
+
+This board utilizes the POSIX arch to (not) simulate the CPU.
```

Comment:
Better would be:

    The nrf52_bsim board definition uses the POSIX architecture to
    run applications natively on the development system.  As with
    the ``native_posix`` board, this has the benefit of providing 
    native code execution performance and easy debugging using
    native tools, but has the same drawbacks.  Please refer to
    :ref:`Native Posix's important limitations <Important_limitations>`
    for more details.